misc/gremlin/Dockerfile

```diff
@@ -10,7 +10,7 @@
 # on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
 # express or implied. See the License for the specific language governing
 # permissions and limitations under the License.
-FROM golang:1.15 as build
+FROM golang:1.15.4 as build
```

Comment:
we an probably get this from the root make file as well, like other dockerfiles do now, right?

Reply:
this has it's own makefile so not sure how passing the var works, but I will try to do that yes

Reply:
done